relay-server/src/actors/project.rs

```diff
@@ -64,6 +64,9 @@ pub struct ProjectConfig {
     /// Configuration for sampling traces, if not present there will be no sampling.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub dynamic_sampling: Option<SamplingConfig>,
+    /// Configuration for operation breakdown. Will be emitted only if present.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub operation_name_breakdown: Option<Vec<String>>,
```

Comment:
```suggestion
    #[serde(skip_serializing_if = "Vec::is_empty")]
    pub operation_name_breakdown: Vec<String>,
```

This way you don't have to unwrap the option, and it remains a noop if there are no breakdowns.